### ISSUE TITLE
track user position on page

### ISSUE THREAD

**Issue by modernNeo** [CSSS/wall_e_website_fe#7]
having the URL track where a user is [i.e. what page they are on and what position is "in-focus"] is a useful way to share their location on the webpage with others, or even just to bookmark for later.

but, say they choose to jump to position 56.

My point of view is that, sure, update windows.location.href so that the page and pos keys accurately reflect where they are, but imo the updating of the windows.location.href should only matter when a user goes to load the page on a fresh page. updates to that aspect should not affect the behaviour of a webpage that is already loaded.

To put it another way, let's see for some reason updating windows.location.href breaks, like chrome pushes an update that breaks that line of code, the only way the wall-e website would be impacted is that the URL doesn't accurately reflect where the user is as they navigate the webpage, but none of the button or navigational functionality on the site itself will be lost.

**Comment by modernNeo** [comment]
moved to https://github.com/CSSS/csss-site-frontend/issues/77
